### ISSUE TITLE
Add a static constructor to instantiate Auth Response from string

### ISSUE THREAD

**Issue by KillDozerX2** [distinction-dev/lambda-authorizer-utils#4]
## Feature Request

Describe the solution you'd like
We should be able to use the parse function and use the methodArn to instantiate an object instead of parsing methodArn first.

For example, currently I have to do this:-
```js
        const arnParts = AuthorizerResponse.parseApiGatewayArn(event.methodArn)
        const response = new AuthorizerResponse(
          'apigateway.amazonaws.com',
          arnParts.region,
          arnParts.awsAccountId,
          arnParts.apiId,
          arnParts.stage
        )
```

And I would like to have a single function to create from methodArn like so

```js
        const response = AuthorizerResponse.fromMethodArn('apigateway.amazonaws.com', event.methodArn)
```

**Comment by KillDozerX2** [comment]
@jappurohit041 this might prove to be even more useful for you.

**Comment by KillDozerX2** [comment]
Solved by #6
